test/metrics_cadvisor.txt

```diff
@@ -907,6 +907,7 @@ container_network_receive_bytes_total{container_name="POD",id="/kubepods/burstab
 container_network_receive_bytes_total{container_name="POD",id="/kubepods/burstable/pod76a4641508128c1971204ecb3dedd60d/d06b05ca98150013ff4f403599bef93f6eb48a5dd52f3f6dc2e28fbbb207e429",image="k8s.gcr.io/pause-amd64:3.0",interface="eth0",name="k8s_POD_kube-scheduler-ip-172-20-57-89.us-east-2.compute.internal_kube-system_76a4641508128c1971204ecb3dedd60d_0",namespace="kube-system",pod_name="kube-scheduler-ip-172-20-57-89.us-east-2.compute.internal"} 1.786404787e+09
 container_network_receive_bytes_total{container_name="POD",id="/kubepods/burstable/podb6fbcc401119c99e5f1b51db3f9db506/1e2b5ffcf815cb12c797719af078f469e4a88da09e1f47138e977d77b2ab2e01",image="k8s.gcr.io/pause-amd64:3.0",interface="eth0",name="k8s_POD_etcd-server-events-ip-172-20-57-89.us-east-2.compute.internal_kube-system_b6fbcc401119c99e5f1b51db3f9db506_0",namespace="kube-system",pod_name="etcd-server-events-ip-172-20-57-89.us-east-2.compute.internal"} 1.786399729e+09
 container_network_receive_bytes_total{container_name="POD",id="/kubepods/burstable/poddc5e34d8b875a92be236483682c1e45b/12e31f62189efe783bc2bb56056bb653fe866ac8f04fd51d13a73984dea14a40",image="k8s.gcr.io/pause-amd64:3.0",interface="eth0",name="k8s_POD_kube-apiserver-ip-172-20-57-89.us-east-2.compute.internal_kube-system_dc5e34d8b875a92be236483682c1e45b_0",namespace="kube-system",pod_name="kube-apiserver-ip-172-20-57-89.us-east-2.compute.internal"} 1.786393994e+09
+container_network_receive_bytes_total{container_name="etcd-container",id="/kubepods/burstable/pod310ff556af68b46e5580c376563548b3/60adb3f0b9e38f68e93d3c7e5d3fa50341d868127c7f4b659e560ed41dad00container_network_receive_errors_total4a",image="k8s.gcr.io/etcd@sha256:19544a655157fb089b62d4dac02bbd095f82ca245dd5e31dd1684d175b109947",name="k8s_etcd-container_etcd-server-ip-172-20-57-89.us-east-2.compute.internal_kube-system_310ff556af68b46e5580c376563548b3_0",namespace="kube-system",pod_name="etcd-server-ip-172-20-57-89.us-east-2.compute.internal",tcp_state="close"} 1.786399238e+09
```

Comment:
Since we were ignoring metric with empty container_name. some metric for container weren't available in `hash_map_cadvisor` hash in test file. So added sample metric